Documentation/proxy.md

```diff
@@ -12,4 +13,4 @@ connection before giving it to gRPC.
 
 If the default proxy doesn't work for you, replace the default dialer with your
 custom proxy dialer. This can be done using
-[`WithDialer`](https://godoc.org/google.golang.org/grpc#WithDialer).
\ No newline at end of file
+[`WithDialer`](https://godoc.org/google.golang.org/grpc#WithDialer).
```

Comment:
What is the diff here lol?

Reply:
Is there a whitespace at the end of the line that the diff viewer is hiding or something? I'm not sure; I didn't touch it but my editor must have changed something? It goes away when hiding whitespace diffs.